av.c

```diff
@@ -466,24 +466,22 @@ Perl_newAVav(pTHX_ AV *oav)
 {
     PERL_ARGS_ASSERT_NEWAVAV;
 
-    if(UNLIKELY(!oav))
+    U32 count = av_count(oav);
```

Comment:
Not that you introduced it, but this should be `Size_t` and not `U32`.  Similarly below.

Reply:
Thanks Tony. Tweaked, squashed, pushed.